crates/bevy_ecs/src/schedule/schedule.rs

```diff
@@ -291,6 +300,32 @@ impl Chain {
 ///     schedule.run(&mut world);
 /// }
 /// ```
+///
+/// Schedules are often inserted into a [`World`] and identified by their [`ScheduleLabel`] only:
+///
+/// ```
+/// # use bevy_ecs::prelude::*;
+/// use bevy_ecs::schedule::ScheduleLabel;
+///
+/// // Declare a new schedule label.
+/// #[derive(ScheduleLabel, Clone, Debug, PartialEq, Eq, Hash, Default)]
+/// struct Update;
```

Comment:
Nit: might be a bit confusing that this is using the name of one of Bevy's default schedules.

Reply:
True. Can you think of a good new example system name? I couldn't, at the time. …maybe something like “`Synchronize`”.